packages/remix/src/build-vite.ts

```diff
@@ -118,18 +119,7 @@ export const build: BuildV2 = async ({
 
   // Determine the version of Remix based on the `@remix-run/dev`
   // package version.
-  const remixRunDevPath = await ensureResolvable(
-    entrypointFsDirname,
-    repoRootPath,
-    '@remix-run/dev'
-  );
-  const remixRunDevPkg = JSON.parse(
-    readFileSync(join(remixRunDevPath, 'package.json'), 'utf8')
-  );
-  const remixVersion = remixRunDevPkg.version;
-
-  // Make `remix build` output production mode
-  spawnOpts.env.NODE_ENV = 'production';
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (blocking):** ‏Do we not need to set `NODE_ENV`?

Reply:
I'm confirming with Mark, but I believe it's no longer necessary.

Reply:
Confirmed.